main.go

```diff
@@ -52,7 +53,17 @@ func main() {
 		}
 	}
 
-	is, err := imagestore.NewImageStore(isoeditor.NewEditor(Options.DataDir), Options.DataDir, versions)
+	var insecureSkipVerify bool
+	insecureSkipVerifyOption := Options.InsecureSkipVerify
+	if insecureSkipVerifyOption == "" || insecureSkipVerifyOption == "false" {
+		insecureSkipVerify = false
+	} else if insecureSkipVerifyOption == "true" {
+		insecureSkipVerify = true
+	} else {
+		log.Fatalf("Unrecognizable option for InsecureSkipVerify.")
+	}
```

Comment:
This can all be removed if the option is a bool.

Reply:
Removed it, thank you!